templates/zerver/help/import-data-from-slack.md

```diff
@@ -25,3 +25,21 @@ If the users are not from the same organization, you should change your organiza
 {!admin.md!}
 
 2. Disable the **Restrict new users to the following email domains** option.
+
+## Elements that are not mapped one to one
+
+- Message edit history. We only keep the latest revision of an edited message.
+- Simultaneous bold and italic formatting of a word. This is not yet supported
+  by Zulip's backend markdown.
+- Permission hierarchy. They are mapped as follows
+  * `Primary owner` and `owner` are mapped to `realm admin`.
+  * `Admin` is mapped to `staff`.
+  * `Member`, `restricted`, and `ultra restricted` are mapped to regular user.
+- Reactions. This is not yet implemented by the conversion script.
+- Gravatar. This is not yet implemented by the conversion script.
```

Comment:
this should just be "Avatar", right?

Reply:
@rht gravatar are already being mapped. So this should be avatars.

Reply:
I looked at https://github.com/zulip/zulip/blob/master/zerver/lib/slack_data_to_zulip_data.py#L62-L64, the gravatar associated with the Slack account is not yet mapped to the gravatar associated with the new Zulip account.

Reply:
The Gravtars are linked with the email id of the user, Hence, the Gravatars will be automatically be displaced if the linked email is used. That's why we are just mapping `avatar_source = 'G'`

Reply:
Huh ic, that's convenient.

Reply:
I will just say non-Gravatar-based avatars to be specific then.